.buildkite/pipelines/e2e.yml

```diff
@@ -0,0 +1,36 @@
+steps:
+  - command: .buildkite/scripts/lifecycle/pre_build.sh
+    label: Pre-Build
+    timeout_in_minutes: 10
+    agents:
+      queue: kibana-default
+    retry:
+      automatic:
+        - exit_status: '*'
+          limit: 1
+
+  - command: .buildkite/scripts/steps/build_kibana.sh
+    label: Build Kibana Distribution and Plugins
+    agents:
+      queue: n2-16-spot
+    key: build
+    if: "build.env('KIBANA_BUILD_ID') == null || build.env('KIBANA_BUILD_ID') == ''"
+    timeout_in_minutes: 60
+    retry:
+      automatic:
+        - exit_status: '-1'
+          limit: 3
+
+  - command: .buildkite/scripts/steps/functional/security_solution.sh
+    label: 'Security Solution Tests'
+    agents:
+      queue: n2-4-spot
+    depends_on: build
+    timeout_in_minutes: 120
+    parallelism: 4
+    retry:
```

Comment:
Can we also post the failures of that pipeline into a slack channel called `#kibana-unsupported-ftrs-alerts`

Reply:
This will need to be updated in the ci repo.

Reply:
we will have to create a follow up for it then 👍